### PR TITLE
Added try-except

### PR DESCRIPTION
**Pull request by T1duS**
Added try-except so that it shows "Nothing playing at the moment" instead of crashing

### CODE REVIEW

File: swaglyrics/spotify.py [hunk 1]
```diff
@@ -30,35 +30,41 @@ def find_spotify_uwp(hwnd, windows):
...
-	session_bus = dbus.SessionBus()
-	spotify_bus = session_bus.get_object("org.mpris.MediaPlayer2.spotify",
-										"/org/mpris/MediaPlayer2")
-	spotify_properties = dbus.Interface(spotify_bus,
-	                                    "org.freedesktop.DBus.Properties")
-	metadata = spotify_properties.Get("org.mpris.MediaPlayer2.Player", "Metadata")
-	track = str(metadata['xesam:title'])
-	artist = str(metadata['xesam:artist'][0])
-	return artist, track
+	try:
```
Comment (aadibajpai) [inline swaglyrics/spotify.py]: Too broad. Narrow it down please :)

File: swaglyrics/spotify.py [hunk 2]
```diff
@@ -30,35 +30,41 @@ def find_spotify_uwp(hwnd, windows):
...
-			set currentTrack to name of current track as string
-			return {currentArtist, currentTrack}
-		end tell
-	end getCurrentlyPlayingTrack
-	"""
-	s = NSAppleScript.alloc().initWithSource_(apple_script_code)
-	x = s.executeAndReturnError_(None)
-	a = str(x[0]).split('"')
-	return a[1], a[3]
+	try:
```
Comment (aadibajpai) [inline swaglyrics/spotify.py]: And here. Look at the windows implementation to get an idea

File: swaglyrics/spotify.py [hunk 3]
```diff
@@ -33,10 +33,13 @@ def get_info_linux():
 	import dbus
 
 	session_bus = dbus.SessionBus()
-	spotify_bus = session_bus.get_object("org.mpris.MediaPlayer2.spotify",
-										"/org/mpris/MediaPlayer2")
+	try:
+		spotify_bus = session_bus.get_object("org.mpris.MediaPlayer2.spotify",
+											"/org/mpris/MediaPlayer2")
```
Comment (aadibajpai) [inline swaglyrics/spotify.py]: why does it need to be separate?

File: swaglyrics/spotify.py [hunk 4]
```diff
@@ -33,10 +33,13 @@ def get_info_linux():
...
+	except:
+		return None
 	spotify_properties = dbus.Interface(spotify_bus,
-	                                    "org.freedesktop.DBus.Properties")
+					    "org.freedesktop.DBus.Properties")
```
Comment (aadibajpai) [inline swaglyrics/spotify.py]: and here

**Comment by T1duS** [comment]
Since there are try-excepts in song, artist functions anyways so there won't be need of them in get_info error handling

**Comment by aadibajpai** [comment]
yes but it's still better to do it in the get_info functions for completeness

**Pull request closed without merging**